src/model.ts

```diff
@@ -1014,10 +1020,20 @@ export class GitExtension implements IGitExtension {
   private async _getServerRoot(): Promise<string> {
     try {
       const response = await httpGitRequest('/git/server_root', 'GET', null);
+      if (response.status === 404) {
+        throw new ServerConnection.ResponseError(
+          response,
+          'Required Git serverextension not available.'
```

Comment:
What about expanding the error message a bit to provide guidance as to next steps?

```suggestion
          'Git server extension is unavailable. Please ensure you have installed the JupyterLab Git server extension by running: pip install --upgrade jupyterlab-git. To confirm that the server extension is installed, run: jupyter serverextension list.'
```

Reply:
Perfect, updated the message.